spec/controllers/application_controller_spec.rb

```
@@ -0,0 +1 @@
+ 
```

rubocop

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)